acceptance-tests/src/main/java/me/atam/atam4jsampleapp/ApplicationConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package me.atam.atam4jsampleapp;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.Configuration;
+
+public class ApplicationConfiguration extends Configuration {
+
+    @JsonProperty
+    private int initialDelayInMillis;
```

Comment:
`initialDelayInMillis` is specified here but not defined in the corresponding `atam4j/acceptance-tests/src/main/resources/app-config.yml` which is a little confusing. Did you forget something here @phillbarber ?